internal/pkg/cli/dialog/use_template.go

```diff
@@ -372,6 +372,9 @@ func (d *useTmplInputsDialog) askInput(inputDef *input.Input) error {
 		}
 		// Save value
 		return d.addInputValue(selectedValues, inputDef, true)
+	case input.KindOAuth:
+		// Save value
+		return d.addInputValue(nil, inputDef, true)
```

Comment:
Tohle už je vlastně věc https://keboola.atlassian.net/browse/KAC-148, ale dávám to sem kvůli e2e testu